src/jit/compiler.hpp

```diff
@@ -764,11 +764,27 @@ inline
 
 inline
 float               getR4LittleEndian(const BYTE * ptr)
-{ return *(UNALIGNED float*)ptr; }
+{
+#ifdef FEATURE_NO_FLOAT_UNALIGNED_ACCESS
```

Comment:
I think it would be better to fix this without adding new #define - make the same code work well enough for all compilers. Something like:

__int32 value = getI4LittleEndian(ptr);
return *(float *)&value;